doc/source/whatsnew/v0.21.0.txt

```diff
@@ -415,6 +415,7 @@ I/O
 - Bug in :func:`read_stata` where value labels could not be read when using an iterator (:issue:`16923`)
 - Bug in :func:`read_html` where import check fails when run in multiple threads (:issue:`16928`)
 - Bug in :func:`read_csv` where automatic delimiter detection caused a ``TypeError`` to be thrown when a bad line was encountered rather than the correct error message (:issue:`13374`)
+- Bug in ``LatexFormatter.write_result`` where repeated multi-index values were not printed even though a higher level index differed from the previous row (:issue:`14484`)
```

Comment:
Reference the function that users will care about, which is "to_latex" instead of "LatexFormatter.write_result" .  Also, follow the convention used in the two lines above when referencing "to_latex" with the `func` notation.